common/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -59,7 +59,7 @@ message GetUsedBytesPResponse {
 }
 
 message WorkerInfo {
-  optional int64 id = 1;
+  optional int64 id = 1 [deprecated = true]; // deprecated in favor of WorkerIdentity
```

Comment:
@jja725 do you know if any code related to PDDM or job service is still relying on this numeric worker ID? Can we have a list of items that need to be migrated to the new worker identity so that this field can be removed entirely?

Reply:
no, this can be removed since mostly we rely on workerNetAddress